Marlin/Configuration.h

```diff
@@ -2823,6 +2823,8 @@
 
 //
 // BigTreeTech Mini 12864 V1.0 is an alias for FYSETC_MINI_12864_2_1. Type A/B. NeoPixel RGB Backlight.
+// If the display lights up but does not show characters, edit the parameter in the file: Marlin\src\lcd\dogm\marlinui_DOGM.cpp
+// change parameter: void MarlinUI::_set_contrast() { u8g.setContrast(contrast); } change to: void MarlinUI::_set_contrast() { u8g.setContrast(255); }
```

Comment:
Resetting EEPROM should set the contrast correctly, so you shouldn’t need to edit this file.

Reply:
I don't think the comment will hurt anyone. But I will remove it well. It works as you say.

Reply:
We'll undoubtedly get a support request from someone trying to modify core Marlin files, so resetting EEPROM is prefered.

Vendors like Fysetc, MKS, and BTT have that blurb in their LCD instruction manual to force the contrast to the max setting which skips the EEPROM reset step.

Reply:
Ok